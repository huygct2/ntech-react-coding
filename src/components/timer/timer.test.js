import React from 'react';
import renderer from 'react-test-renderer';
import Timer from './timer';

test('Timer component', () => {
  const component = renderer.create(
    <Timer time='2018-02-12T11:00:00.000Z' />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
