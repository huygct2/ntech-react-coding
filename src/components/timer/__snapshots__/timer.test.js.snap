// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer component 1`] = `
<time>
  Starts in 
  <span>
    <span
      className="amount-label"
    >
      2
    </span>
    <span
      className="unit-label"
    >
      h
       
    </span>
  </span>
</time>
`;
